Origin: https://sources.debian.org/src/socat/1.8.0.2-1/debian/patches/01-Index/
Source-package: socat_1.8.0.2-1.debian.tar.xz (SHA256 04b361cd91884eda6ca9fe2890ca83a6abe73dd8ce75fe9927b2d8443af36123)

Generate index page for doc-base compatibility.

Forwarded: no
Author: Chris Taylor <ctaylor@debian.org>
Date: 10FEB2010
--- /dev/null
+++ b/doc/index.html
@@ -0,0 +1,14 @@
+<html>
+<head>
+<title>Index of socat online documentation</title>
+</head>
+<body>
+<div align=left>
+<h1>Index of socat online documentation</h1>
+<a href="socat.html">Socat Manual Page</a><br><br>
+<a href="socat-genericsocket.html">Generic sockets with Socat</a><br><br>
+<a href="socat-multicast.html">IP Multicasting with Socat</a><br><br>
+<a href="socat-openssltunnel.html">Securing Traffic Between two Socat Instances Using SSL</a><br><br>
+<a href="socat-tun.html">Building TUN based virtual networks with soca</a><br><br>
+</body>
+</html>
